LrcSubPub:
  xavier_to_lrc:
    topic: /xav2lrc_msg
    queue_size: 1
  ocr_to_lrc:
    topic: /ocr2lrc_msg
    queue_size: 1
  lrc_to_xavier:
    topic: /lrc2xav_msg
    queue_size: 1
  lrc_to_ocr:
    topic: /lrc2ocr_msg
    queue_size: 1
  speed:
    topic: /carla/ego_vehicle/vehicle_status
    queue_size: 1

LrcParams:
  udp_group_addr: 239.255.255.250
  udp_group_port: 9392
  lrc_log_path: /home/jetson/catkin_ws/logfiles/
  epsilon: 0.7
  lu_ob_A: 0.6817
  lu_ob_B: 0.3183
  lu_ob_L: 0.2817
  enable_console_output: true
